File: Aula_0527/lista03ex02.java
class Frete {
  private double distancia, peso;
  public Frete(double distancia, double peso) {
    this.distancia = distancia;
    this.peso = peso;
  }
  public double GetDistancia() {
    return distancia;
  }
  public double GetPeso() {
    return peso;
  }
  public String toString() {
    return String.format("Distancia = %f km, Peso = %f kg", distancia, peso);
  }
  public void SetDistancia(double distancia) {
    this.distancia = distancia;
  }
  public void SetPeso(double peso) {
    this.peso = peso;
  }
  public double CalcFrete() {
    return 0.01 * distancia * peso;
  }
}

class Program {
  public static void main(String[] args) {
    Frete f = new Frete(100, 10);
    System.out.println(f);
    System.out.println(f.GetDistancia());
    System.out.println(f.GetPeso());
    System.out.println(f.CalcFrete());
    f.SetDistancia(150);
    System.out.println(f);
    System.out.println(f.CalcFrete());
  }
}

File: Aula_0527/lista03ex02.py
class Frete:                           
  def __init__(self, distancia, peso):
    self.__distancia = distancia
    self.__peso = peso
  def get_distancia(self):
    return self.__distancia
  def get_peso(self):
    return self.__peso
  def set_distancia(self, distancia):
    self.__distancia = distancia
  def set_peso(self, peso):
    self.__peso = peso
  def calc_frete(self):
    return 0.01 * self.__distancia * self.__peso
  def __str__(self):
    return f"Distancia = {self.__distancia} km, Peso = {self.__peso} kg"


if __name__ == "__main__":
  f = Frete(100, 10)  
  print(f)
  print(f.get_distancia())
  print(f.get_peso())
  print(f.calc_frete())
  f.set_distancia(150)
  print(f)
  print(f.calc_frete())

